mi/core/instrument/publisher.py

```diff
@@ -140,7 +140,7 @@ def filter_events(self, events):
         return events
 
     @staticmethod
-    def from_url(url, headers=None, allowed=None, max_events=None, **kwargs):
+    def from_url(url, handler=None, headers=None, allowed=None, max_events=None, **kwargs):
```

Comment:
Wouldn't it be safer to add "handler" to the end of the list to make it backward compatible in case there's code out there like the original call in Playbackwrapper with "headers" in the second position?